mlos_bench/mlos_bench/services/local/local_exec.py

```diff
@@ -195,8 +195,10 @@ def _local_exec_script(self, script_line: str,
         cmd = [token for subcmd in subcmds for token in subcmd]
 
         env: Dict[str, str] = {}
+        # Need to include at least some basic environment variables to run the script.
+        env["PATH"] = environ["PATH"]
```

Comment:
A small bug I noticed while debugging this.  Could be expanded to additional items, but this was the minimal set necessary for shebang scripts to work.